src/Sylius/Bundle/CoreBundle/CatalogPromotion/Validator/Constraints/CatalogPromotionScopeValidator.php

```diff
@@ -11,9 +11,9 @@
 
 declare(strict_types=1);
 
-namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
+namespace Sylius\Bundle\CoreBundle\CatalogPromotion\Validator\Constraints;
```

Comment:
I wonder, can we move it to *PromotionBundle*? It's almost identical, we could maybe even unify these services and register them under 2 different names for 2 different tags

Reply:
It also somehow completes [this PR](https://github.com/Sylius/Sylius/pull/13627)

Reply:
Almost identical to `CatalogPromotionActionValidator`, sorry, I've lost the part of the sentence 🖖 I would probably be for moving all of these into the **PromotionBundle**, as this functionality is not related to products or variants at all (specific validators are) and drop the validation messages in constraint and just use translation keys in the validators (we're anyway doing this e.g. [here](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Validator/CatalogPromotionAction/FixedDiscountActionValidator.php#L50)). But I agree it can be done in the separate PR 🚀